syntax = "proto3";

package feb_stats;

service FebStatsService {
    rpc GetFebStats (GetFebStatsRequest) returns (GetFebStatsResponse);
}

message GetFebStatsRequest {
    repeated bytes boxscores = 1;
    bool color_sheet = 2;
}

message GetFebStatsResponse {
    bytes sheet = 1;
    repeated string teams = 2;
}